next/components/Molecules/Carousel/SliderControl/SliderControl.tsx

```diff
@@ -0,0 +1,34 @@
+import ChevronLeft from '@assets/images/chevron-left.svg'
+import ChevronRight from '@assets/images/chevron-right.svg'
+import VerticalCardButton from '@components/HomePage/VerticalCardButton'
+import cx from 'classnames'
+import React from 'react'
+
+export enum SliderVariant {
+  LEFT,
+  RIGHT,
+}
+
+interface SliderControlProps {
+  variant: SliderVariant
```

Comment:
Just naming again 😇 
We usually use `variant` for component styles, see e.g. `<Button variant="plain-primary">`.

Here I see opportunity to use for example `direction` and `SliderDirection` which says more what it really does.